src/journal-remote/headers-util.c

```diff
@@ -0,0 +1,207 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
This file is mostly a subset of env-util.c .
Please try to use env-util.c .

Reply:
it's not a subset, it has differences